examples/video-chat/package.json

```diff
@@ -8,14 +8,17 @@
     "ios": "npx expo run:ios -d",
     "start": "npx react-native start",
     "test": "jest",
-    "format:fix": "prettier --write . --ignore-path ./.eslintignore",
+    "build": "tsc",
+    "open:ios": "open -a \"Xcode\" ios/FishjamExample.xcworkspace",
+    "open:android": "open -a \"Android Studio\" android",
+    "format": "prettier --write . --ignore-path ./.eslintignore",
     "format:check": "prettier --check . --ignore-path ./.eslintignore",
-    "lint:fix": "eslint . --ext .ts,.tsx --fix",
+    "lint": "eslint . --ext .ts,.tsx --fix",
     "lint:check": "eslint . --ext .ts,.tsx"
   },
   "dependencies": {
     "@expo-google-fonts/noto-sans": "^0.2.3",
-    "@fishjam-dev/react-native-client": "link:..",
+    "@fishjam-dev/react-native-client": "*",
```

Comment:
why * instead of link?
this link is very useful when developing native modules - so you can modify the files in src and rebuild the app with the edited code

Reply:
This is how yarn workspaces works. By using `*` you are telling yarn, that this dependency is part of workspace. And all workspaces are defined in main `package.json` file. Native dependencies are still linked via path (that is why there are new `../` in paths in gradle/podfile).
But TL;DR; it will work as `link` before, but better (single `node_moduels` directory and easier dependency managment)